### DISCUSSION TITLE
Generating C/C++ code with intrinsics

### DISCUSSION THREAD

**Discussion by zzy-supplient** [ispc/ispc#2644] [General]
Sometimes it can be helpful to manually optimize the generated code.

In issue #92 , commit 8938e14 seems to have given a solution. However I cannot find the switch `--emit-c++` in v1.21.0.

**Comment by nurmukhametov** [comment]
"generic" targets were removed in https://github.com/ispc/ispc/commit/78502814558d5915afa00540fff0a683cbee92be, so there is no `--emit-c++` switch since v1.14.0

**Reply by imaginary-person** [reply]
That's highly unfortunate
